reportportal_client/client.pyi

```diff
@@ -1,57 +1,26 @@
-from requests import Response
-from typing import Any, Text, Optional
+from reportportal_client.core.test_manager import TestManager as TestManager
+from reportportal_client.helpers import dict_to_payload as dict_to_payload, get_id as get_id, get_msg as get_msg, uri_join as uri_join
+from typing import Any, Dict, List, Optional, Text
+from requests import Session
+
+logger: Any
 
 class RPClient:
-    _launch_uuid: Optional[Text] = ...
-    _token: Optional[Text] = ...
-    api_version: Text = ...
-    base_url: Text = ...
-    password: Text = ...
-    port: Optional[Text] = ...
-    project_name: Text = ...
-    username: Text = ...
-    def __init__(
-            self,
-            base_url: Text,
-            username: Text,
-            password: Text,
-            project_name: Text,
-            api_version: Text,
-            **kwargs: Any) -> None: ...
-    @property
-    def launch_uuid(self) -> Text: ...
-    @property
-    def token(self) -> Text: ...
-    def _request(self,
-                 uri: Text,
-                 token: Text,
-                 **kwargs: Any) -> Response: ...
-    def start_launch(
-            self,
-            name: Text,
-            start_time: Text,
-            **kwargs: Any) -> None: ...
-    def start_item(
-            self,
-            name: Text,
-            start_time: Text,
-            item_type: Text,
-            launch_uuid: Text,
-            parent_uuid: Optional[Text],
-            **kwargs: Any) -> None: ...
-    def finish_item(
-            self,
-            launch_uuid: Text,
-            item_uuid: Text,
-            end_time: Text,
-            **kwargs: Any) -> None: ...
-    def finish_launch(
-            self,
-            launch_uuid: Text,
-            end_time: Text,
-            **kwargs: Any) -> None: ...
-    def save_log(
-            self,
-            launch_uuid: Text,
-            log_time: Text,
-            **kwargs: Any) -> None: ...
+    endpoint: Text = ...
+    log_batch_size: int = ...
+    project: Text = ...
+    token: Text = ...
+    launch_id: Optional[Text] = ...
+    verify_ssl: bool = ...
+    is_skipped_an_issue: bool = ...
+    api_v1: str = ...
+    api_v2: str = ...
+    base_url_v1: Text = ...
+    base_url_v2: Text = ...
+    session: Session = ...
+    def __init__(self, endpoint: str, project: str, token: str, log_batch_size: int=..., is_skipped_an_issue: bool=..., verify_ssl: bool=..., retries: Optional[int]=..., max_pool_size: int=..., launch_id: Optional[str]=...) -> None: ...
```

Comment:
Please, use indentation in .pyi files as well.

Reply:
Done